ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-oadp-1.4.yaml

```diff
@@ -0,0 +1,40 @@
+build_root:
```

Comment:
We can omit everything related to the CSI plugin, since this has been moved into the main velero tree for Velero 1.14

Reply:
The CSI config and jobs should be gone from 1.4 now.